app/views/group/index.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="d-grid justify-content-center">
```

Comment:
- [ ] It would be better if we display the total amount of the category in the top part of the page. This will help sum all the amount for us

![Screenshot from 2022-09-08 18-37-03](https://user-images.githubusercontent.com/46388113/189202402-24f73da7-8e00-40b0-a074-03c55c727060.png)

Reply:
Done